packages/@lwc/features/src/__tests__/flags-prod.spec.ts

```diff
@@ -1,268 +0,0 @@
-                    }
-                }
-                if (features.ENABLE_FEATURE_TRUE) {
-                    if (features.ENABLE_FEATURE_NULL) {
-                        console.log('nested feature flags sounds like a vary bad idea');
-                    }
-                }
-            `,
-            output: `
-                import features, { lwcRuntimeFlags } from '@lwc/features';
-                if (lwcRuntimeFlags.ENABLE_FEATURE_NULL) {
-                    {
-                        console.log('nested feature flags sounds like a vary bad idea');
-                    }
-                }
-                {
-                    if (lwcRuntimeFlags.ENABLE_FEATURE_NULL) {
-                        console.log('nested feature flags sounds like a vary bad idea');
-                    }
-                }
-            `,
-        },
-        'should not transform tests that are not an actual reference to the imported binding': {
-            code: `
-                import featureFlag from '@lwc/features';
-                function awesome() {
-                    const featureFlag = { ENABLE_FEATURE_FALSE: false };
-                    if (featureFlag.ENABLE_FEATURE_FALSE) {
-                        console.log('featureFlag.ENABLE_FEATURE_FALSE');
-                    }
-                    if (!featureFlag.ENABLE_FEATURE_FALSE) {
-                        console.log('!featureFlag.ENABLE_FEATURE_FALSE');
-                    }
-                }
-            `,
-            output: `
-                import featureFlag, { lwcRuntimeFlags } from '@lwc/features';
-                function awesome() {
-                    const featureFlag = {
-                        ENABLE_FEATURE_FALSE: false,
-                    };
-                    if (featureFlag.ENABLE_FEATURE_FALSE) {
-                        console.log('featureFlag.ENABLE_FEATURE_FALSE');
-                    }
-                    if (!featureFlag.ENABLE_FEATURE_FALSE) {
-                        console.log('!featureFlag.ENABLE_FEATURE_FALSE');
-                    }
-                }
-            `,
-        },
-        'should not transform member expressions that are not runtime flag lookups': {
-            code: `
-                import { lwcRuntimeFlags } from '@lwc/features';
-                if (churroteria.ENABLE_FEATURE_TRUE) {
-                    console.log('churroteria.ENABLE_FEATURE_TRUE');
-                }
-            `,
-            output: `
-                import { lwcRuntimeFlags } from '@lwc/features';
-                if (churroteria.ENABLE_FEATURE_TRUE) {
-                    console.log('churroteria.ENABLE_FEATURE_TRUE');
-                }
-            `,
-        },
-    },
-});
-
-// Forces TypeScript to treat this file as a module, not a script, which avoids:
-// Error TS2451: Cannot redeclare block-scoped variable 'pluginTester'
-// ...which are due to the same top-level variable names being used here and in flags.spec.ts.
-export {};
```

Comment:
All these Jest tests just tested the Babel plugin. See #3305 for some new Jest tests to verify the functional behavior of `setFeatureFlag`.